source/conf.py

```diff
@@ -49,14 +49,9 @@
 rst_prolog = """
 .. |PRODUCT| replace:: ``Rsyslog``
 
-.. |FormatsBasicOldName| replace:: ``sysklogd``
-.. |FormatsBasicNewName| replace:: ``basic``
-
-.. |FormatsAdvancedOldName| replace:: ``RainerScript``
-.. |FormatsAdvancedNewName| replace:: ``advanced``
-
-.. |FormatsObsoleteOldName| replace:: ``legacy``
-.. |FormatsObsoleteNewName| replace:: ``obsolete legacy``
+.. |FmtBasicNewName| replace:: ``basic``
```

Comment:
sorry for being insistent... do we need "New" inside the name? ```FmtBasicName``` would be good IMHO...

Reply:
No, we don't; good catch. I think I meant to do that based on your prior suggestion, but forgot about it when I made this quick change.

Provide via a separate commit or squash with the previous one?